### DISCUSSION TITLE
Plot directions on a stereographic triangle for a cubic crystal

### DISCUSSION THREAD

**Discussion by starmm11** [mtex-toolbox/mtex#1446] [Ask Anything] [answered]
Hi, 

I am a new mtex user. Can you suggest how to visualize directions (points) on a stereographic triangle 
for cubic crystals with vertices in [001][110][111]? I have only seen examples (in Pole figures) where these points 
are shown on the entire steographic projection of the sphere.

Many thanks

**Comment by kilir** [comment, marked as answer]
Hi,
potentially something like that?
```
cs = crystalSymmetry('m-3m')
h = Miller.rand(10,cs)
plot(h.project2FundamentalRegion,'fundamentalRegion')
```
Cheers,
Rüdiger

**Reply by starmm11** [reply]
Wow! That works better than I expected! Thanks a lot!
